practices/dependency-management.md

```diff
@@ -0,0 +1,104 @@
+# Dependency management
+
+The following guide describes how we manage package dependencies in our Node.js-based projects.
+
+
+## Specifying versions of dependencies
+
+As a general rule, we tend to follow a conservative approach when specifying versions for the dependencies of our packages. This helps us ensure consistency between different environments, as we can't know for sure in what environment the app is going to be running, and prevent breakage caused by major or minor updates in dependencies.
```

Comment:
It would be helpful to clarify - here - what we mean by "a conservative approach". Saying something along the lines of "we only allow PATCH version ranges for run-time dependencies, and … etc etc" will help people …